R/data.R

```diff
@@ -1,510 +1,877 @@
+#' A data frame with 19 rows and 5 columns.
+#' \describe{
+#'   \item{chrom}{Chromosomes without chr-prefix, 1:22.}
+#'   \item{start}{Start coordinate for the specified oncogene.}
+#'   \item{end}{End coordinate for the specified oncogene.}
+#'   \item{gene}{Lymphoma oncogene.}
+#'   \item{entrez}{ENTREZ ID for the specified oncogene.}
+#' }
+"grch37_oncogene"
+
+#' Oncogenes in hg38 genome build.
+#'
+#' A data frame with the coordinates of lymphoma oncogenes relative to the hg38 genome build.
+#'
+#' @format ## `hg38_oncogene`
+#' A data frame with 19 rows and 5 columns.
+#' \describe{
+#'   \item{chrom}{Chromosomes without chr-prefix, 1:22.}
+#'   \item{start}{Start coordinate for the specified oncogene.}
+#'   \item{end}{End coordinate for the specified oncogene.}
+#'   \item{gene}{Lymphoma oncogene.}
+#'   \item{entrez}{ENTREZ ID for the specified oncogene.}
+#' }
+"hg38_oncogene"
+
+#' Chromosome Arms grch37.
+#'
+#' A data frame with the chromosome arm coordinates in respect to grch37.
+#'
+#' @format ## `chromosome_arms_grch37`
+#' A data frame with 48 rows and 4 columns.
+#' \describe{
+#'   \item{chromosome}{Chromosomes without chr-prefix, 1:22, X and Y.}
+#'   \item{start}{Start coordinates for the specified chromosome arm.}
+#'   \item{end}{End coordinates for the specified chromosome arm.}
+#'   \item{arm}{Chromosome arm, either p or q.}
+#' }
+"chromosome_arms_grch37"
+
+
+#' Chromosome Arms hg38.
+#'
+#' A data frame with the chromosome arm coordinates in respect to hg38.
+#'
+#' @format ## `chromosome_arms_hg38`
+#' A data frame with 48 rows and 4 columns.
+#' \describe{
+#'   \item{chromosome}{Chromosomes with chr-prefix, 1:22, X and Y.}
+#'   \item{start}{Start coordinates for the specified chromosome arm.}
+#'   \item{end}{End coordinates for the specified chromosome arm.}
+#'   \item{arm}{Chromosome arm, either p or q.}
+#' }
+"chromosome_arms_hg38"
+
+
+#' Double Hit Signature Genes With Weights.
+#'
+#' A data frame with double hit signature genes (both as ensembl IDs and Hugo symbols) and importance scores.
+#'
+#' @format ## `dhitsig_genes_with_weights`
+#' A data frame with 104 rows and 3 columns.
+#' \describe{
+#'   \item{Ensembl_ID}{Ensembl IDs as factors, 104 levels.}
+#'   \item{ImportanceScore}{Numeric column with importance scores.}
+#'   \item{Hugo_Symbol}{Gene symbols in Hugo format as a factor with 104 levels.}
+#' }
+"dhitsig_genes_with_weights"
 
-#' @title Get genes from one or more gene lists
+
+#' Genes Blacklist.
 #'
-#' @description Retrieve gene names from bundled lymphoma gene lists.
+#' A tibble with gene symbols (Hugo) that falls within blacklisted regions of the genome.
 #'
-#' @details Complete lists of genes described as significantly mutated in large
-#' lymphoma studies have been curated and provided with this package.
+#' @format ## `gene_blacklist`
+#' A tibble with 291 rows.
+#' \describe{
+#'   \item{Gene}{Genes symbols in Hugo format.}
+#' }
+"gene_blacklist"
+
+
+#' Grande et al. MAF.
```

Comment:
The grande_maf should be deprecated now, since we have that sample_data now 😄

Reply:
For sure, I just migrated all the datasets in GAMBLR main repo, so we would only have to remove any unwanted objects (like this one). I'll address this in a commit along with the other items you pointed out. Thanks!